.github/workflows/test-pr.yml

```diff
@@ -36,12 +36,14 @@ jobs:
         run: git push origin HEAD:${GITHUB_HEAD_REF}
 
   kevm-pyk-code-quality-checks:
-    needs: version-bump
     name: 'Code Quality Checks'
-    runs-on: ubuntu-latest
+    runs-on: [self-hosted, linux, flyweight]
```

Comment:
These runners are not ephemeral, are they? If not, we might run into an issue similar to this one: https://github.com/runtimeverification/evm-semantics/issues/1457

Reply:
Sorry I got pulled around too much this week juggling other issues and left this hanging. It got merged. But I will post a new PR testing a move to ephemeral